src/main/java/org/edumips64/ui/swing/DineroFrontend.java

```diff
@@ -46,8 +46,15 @@
   // one instance of DineroFrame will be created in EduMIPS64
   private static final Logger logger = Logger.getLogger(DineroFrontend.class.getName());
   private static JTextField path, params;
-  private static JButton execute;
+  private static JButton execute, configure, create;	//configure and create buttons added to create panels and cache parameters
```

Comment:
It would be nice to not need a `configure` action at all, but rather re-create dynamically the command line every time the user interacts with any of the UI elements.

Note: this is a larger and optional change, feel free to address it in a subsequent PR.

Reply:
I agree. I will address this in another PR and update here later on.